### ISSUE TITLE
SendInBlue Provider Support

### ISSUE THREAD

**Issue by lapubell** [mettle/sendportal#30]
I love me some laravel, and I saw your package from the laraval news email. I'd love to use this for my business, but we use SendInBlue for pretty much all our email communications.

I've already forked the repo, and would love to open a PR once I integrate the provider into the codebase. Any tips while I dive in?

**Comment by JonoB** [comment]
Happy to accept a PR that adds SendInBlue as an Email Service.

Someone else is working on a PR for another provider (ElasticEmailk) that you can use as a reference: https://github.com/mettle/sendportal-core/pull/24

**Comment by mpixelz** [comment]
any luck on this?

**Comment by lapubell** [comment]
Not really, we aren't even sticking with SendInBlue anymore and I see that you already have a driver for Sendgrid.
